sim_defs.h

```diff
@@ -1145,6 +1145,16 @@ struct MEMFILE {
                                                     sim_printf("%s failed at %s line %d\n", #_Expression, __FILE__, __LINE__);       \
                                                     abort();}
 
+#if defined(assert)
+#error "Don't user assert().  Feel free to use ASSURE(expression) in its place."

```

Comment:
A typo, should be `use`

Reply:
Probably makes sense to reword into something like "It's advised to use ASSURE(expression) instead"